omod/src/test/java/org/openmrs/module/sync2/RestResourceConverterTest.java

```diff
@@ -81,5 +77,34 @@ public void convertObject_shouldConvertPersonAttributesToUuids() throws Exceptio
 		assertEquals(1, uuids.size());
 		assertEquals(conceptUuid, uuids.get(0));
 	}
-	
+
+	@Test
+	public void convertObjectShouldConvertEncounterToSuitableRepresentationForRestPost() {
+		SimpleObject encounter = new SimpleObject();
+		final String encounterUuid = "some-encounter-uuid-who-gives-a-damn?";
```

Comment:
Same here in regards to our naming convention for constants. https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions